src/app.js

```diff
@@ -17,16 +17,32 @@ const slack = require('./slackClient');
 const dynamo = require('./dynamo');
 
 async function postEphemeral({channel, text, blocks, user}) {
-	const result = await slack.chat.postEphemeral({channel, text, blocks, user});
-	if (!result.ok) {
-		throw new AppError(`Failed to post message: ${result.error}`, 500);
+	try {
+		const result = await slack.chat.postEphemeral({channel, text, blocks, user});
+		if (!result.ok) {
```

Comment:
Not sure if this is possible but just in case. The docs suggest yes but in practice we've seen exceptions on failures.

https://api.slack.com/methods/chat.postEphemeral#response